themes/vocabulary_theme/templates/layout.html

```diff
@@ -73,12 +71,13 @@
                 ['/contributing-code/github-repo-guidelines', 'GitHub Repo Guidelines'],
                 ['/contributing-code/cc-search', 'CC Search'],
               ] %}
-                <a class="navbar-item" href="{{ href|url }}">{{ title }}</a>
+              <a class="navbar-item" href="{{ href|url }}">{{ title }}</a>
               {% endfor %}
             </div>
           </div>
           <div class="navbar-item has-dropdown is-hoverable">
-            <a class="navbar-link is-arrowless {% if this.is_child_of('/community') %} is-active{% endif%}">Community<i class="icon caret-down"></i></a>
+            <a class="navbar-link is-arrowless {% if this.is_child_of('/community') %} is-active{% endif%}">Community<i
+                class="icon caret-down"></i></a>
```

Comment:
Again, please use linebreaks at appropriate locations.
```suggestion
            <a class="navbar-link is-arrowless {% if this.is_child_of('/community') %} is-active{% endif%}">
              Community <i class="icon caret-down"></i>
            </a>
```